lib/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const MYSQL_CONFIG = {
```

Comment:
I don't think we need a constants file for the env variables - we should just be able to reference the `env`... by doing this we'll have to maintain the variables in two places: here and inside of .env